# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ash/src/vk/macros.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/ash/src/vk/macros.rs b/third_party/rust/ash/src/vk/macros.rs
--- a/third_party/rust/ash/src/vk/macros.rs
+++ b/third_party/rust/ash/src/vk/macros.rs
@@ -1,24 +1,8 @@
-#[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_MAKE_VERSION.html>"]
-pub const fn make_version(major: u32, minor: u32, patch: u32) -> u32 {
-    (major << 22) | (minor << 12) | patch
-}
-#[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_VERSION_MAJOR.html>"]
-pub const fn version_major(version: u32) -> u32 {
-    version >> 22
-}
-#[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_VERSION_MINOR.html>"]
-pub const fn version_minor(version: u32) -> u32 {
-    (version >> 12) & 0x3ff
-}
-#[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/VK_VERSION_PATCH.html>"]
-pub const fn version_patch(version: u32) -> u32 {
-    version & 0xfff
-}
 #[macro_export]
 macro_rules! vk_bitflags_wrapped {
     ($ name : ident , $ all : expr , $ flag_type : ty) => {
         impl Default for $name {
             fn default() -> $name {
                 $name(0)
             }
         }